templates/404.html

```diff
@@ -4,15 +4,15 @@
 
 {% block header %}
 
-<div class="p-strip">
-  <div class="row">
-    <div class="u-equal-height u-vertically-center">
-      <div class="col-6">
-        <img src="/static/img/image-404.svg" alt="Error owl" />
-      </div>
-      <div class="col-6">
-        <h1>404: Page not found.</h1>
-        <p>Sorry, we couldn&rsquo;t find that page</p>
+<div class="p-strip is-deep">
+  <div class="row u-equal-height">
+    <div class="col-6 u-vertically-center u-align--center">
+        <img src="/static/img/image-404.svg" alt="Error owl" style="height: 365px; width:360px"/>
```

Comment:
Hi @minkyngkm ! Just had a question about the image, is it supposed to have the `hi_def` and `loading` attributes as well? I couldn't find it on manager.assets so I'm not sure

Reply:
Actually, I think iI found it. Could it might be this one?
```
          {{ image (
            url="https://assets.ubuntu.com/v1/03c7318e-image-404.svg",
            alt="",
            width="442",
            height="436",
            hi_def=True,
            loading="auto|lazy"
            ) | safe
          }}
```

Reply:
Hmmm I thought it's not necessary because the image is not from the server but just to make sure let's ask @bethcollins92. Could you please give us advice Beth?

Reply:
I'm not sure why it was done like this but I think it's fine to leave it, as it's only one image visible on page load so the loading will need to be auto anyway - but if you did want to use the image tag you can copy the one from [ubuntu.com](https://github.com/canonical-web-and-design/ubuntu.com/blob/master/templates/404.html)

Reply:
Ahhh I see. All good then! :)

Reply:
Okay! Thanks for the feedback @bethcollins92 and thanks for reviewing @mtruj013! :)